src/miner.cpp

```diff
@@ -45,7 +45,9 @@ BlockAssembler::Options::Options() {
     nBlockMaxWeight = DEFAULT_BLOCK_MAX_WEIGHT;
 }
 
-BlockAssembler::BlockAssembler(const CChainParams& params, const Options& options) : chainparams(params)
+BlockAssembler::BlockAssembler(const CTxMemPool& mempool, const CChainParams& params, const Options& options)
+    : chainparams(params),
```

Comment:
nit: order member initialization list the same as the parameter list (and preferably on one line)

Reply:
I prefer multiple lines